### PR TITLE
WIP feat: support customizing additional labels of metrics

### PR DESCRIPTION
**Pull request by dmsolr**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

#4273

### CODE REVIEW

File: conf/config-default.yaml [hunk 1]
```diff
@@ -438,6 +438,9 @@ plugin_attr:
     export_addr:
       ip: 127.0.0.1
       port: 9091
+        #    labels:
+        #remote_addr: $remote_addr
+        #request_method: $request_method
```
Comment (zhendongcmss) [inline conf/config-default.yaml:443]: This function is userful for us. we add metrics too on our code as hard code way.
Can you list ot give a link which labels are support ?

File: docs/zh/latest/plugins/prometheus.md [hunk 2]
```diff
@@ -132,6 +132,24 @@ plugin_attr:
     export_uri: /apisix/metrics
 ```
 
+## 如何自定义附加标签
+
+该插件支持在`conf/config`的`plugin_attr`部分自定义度量标签`Status codes`和`Latency`。亚马尔`
```
Comment (starsz) [inline docs/zh/latest/plugins/prometheus.md:137]: 亚马尔 ？？
Maybe `yaml" ?

**Review by starsz — COMMENTED** [review]
Hi, @dmsolr Can you add test cases to cover this good feature?

File: docs/en/latest/plugins/prometheus.md [hunk 3]
```diff
@@ -129,10 +129,28 @@ Here is an example:
 
 ```yaml
 plugin_attr:
-  prometheus:
+prometheus:
```
Comment (leslie-tsang) [inline docs/en/latest/plugins/prometheus.md:132]:
```suggestion
  prometheus:
```
There seems to be a mistake here.

File: docs/en/latest/plugins/prometheus.md [hunk 4]
```diff
@@ -129,10 +129,28 @@ Here is an example:
...
     export_uri: /apisix/metrics
 ```
 
+## How to custom additional labels
+
+The plugin is supported to custom labels for metrics, `Status codes` and `Latency`, in the `plugin_attr` section of `conf/config.yaml`
+
+| Name       | Type   | Default | Description                       |
+| ---------- | ------ | ------- | --------------------------------- |
+| labels     | array  |         | custom labels from [Nginx variables](http://nginx.org/en/docs/varindex.html) |
```
Comment (leslie-tsang) [inline docs/en/latest/plugins/prometheus.md:142]: Better include the references to APISIX build-in variables  as well

File: docs/zh/latest/plugins/prometheus.md (hunk 2 above)
Comment (leslie-tsang) [inline docs/zh/latest/plugins/prometheus.md:137]:
```suggestion
该插件支持在 `conf/config` 的 `plugin_attr` 部分自定义度量标签 `Status codes` 和 `Latency`。
```

File: docs/zh/latest/plugins/prometheus.md [hunk 5]
```diff
@@ -132,6 +132,24 @@ plugin_attr:
...
+
+| 名称       | 类型   | 默认值   | 描述                       
+| --------- | ----- | -------- | ------------------------- |
+| labels    | array |          | 使用 [Nginx 变量](http://nginx.org/en/docs/varindex.html)作为自定义标签 |
```
Comment (leslie-tsang) [inline docs/zh/latest/plugins/prometheus.md:141]: Ditto

**Review by leslie-tsang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**